Modules/TPC/src/CheckOfTrendings.cxx

```diff
@@ -29,41 +30,127 @@
 namespace o2::quality_control_modules::tpc
 {
 
-void CheckOfTrendings::configure() {}
+void CheckOfTrendings::configure()
+{
+  if (auto param = mCustomParameters.find("chooseCheckMeanOrExpectedPhysicsValueOrBoth"); param != mCustomParameters.end()) {
+    checkChoice = param->second.c_str();
+    if (checkChoice != "mean" && checkChoice != "ExpextedPhysicsValue" && checkChoice != "both") {
```

Comment:
Perhaps you should make all these strings (or at least the ones used more than once) `static constexpr std::string_view` here (or in the header if used also in other places) and then use it everywhere to avoid copy'n'paste errors

Reply:
I did it only for strings which are used more than twice